lib/api/unittest/CDataFrameAnalysisSpecificationTest.cc

```diff
@@ -229,16 +229,12 @@ void CDataFrameAnalysisSpecificationTest::testRunAnalysis() {
         CPPUNIT_ASSERT(runner != nullptr);
 
         double lastProgress{runner->progress()};
-        for (;;) {
+        while (runner->finished()) {
```

Comment:
Should it be `== false`?

Reply:
doh! Committing without running the test